crypto/bls.go

```diff
@@ -243,18 +247,7 @@ const identityBLSSignatureHeader = byte(0xC0)
 // suspected to be equal to identity, which avoids failing the aggregated
 // signature verification.
 func IsBLSSignatureIdentity(s Signature) bool {
-	if len(s) != signatureLengthBLSBLS12381 {
-		return false
-	}
-	if s[0] != identityBLSSignatureHeader {
-		return false
-	}
-	for _, b := range s[1:] {
-		if b != 0 {
-			return false
-		}
-	}
-	return true
+	return bytes.Compare(s, identityBLSSignature) == 0
```

Comment:
```suggestion
	return bytes.Equal(s, identityBLSSignature)
```

Reply:
Yep that was also flagged by the linter, but I was waiting for @pattyshack's opinion in case I need to undo all the change.

Reply:
i don't feel strongly about this.  feel free to follow alexey's suggestion